tests/test_extractors.py

```diff
@@ -0,0 +1,99 @@
+import attrs
+import pytest
+from parsel import Selector
+from web_poet import field
+
+from zyte_common_items import (
+    ProductFromList,
+    ProductFromListExtractor,
+    ProductFromListSelectorExtractor,
+    ProductVariant,
+    ProductVariantExtractor,
+    ProductVariantSelectorExtractor,
+)
+
+
+@pytest.mark.asyncio
+async def test_product_from_list_extractor():
+    @attrs.define
+    class MyProductFromListExtractor(ProductFromListExtractor):
+        data: dict
+
+        @field
+        def price(self):
+            return self.data["price"]
+
+        @field
+        def regularPrice(self):
+            return self.data["regularPrice"]
+
+    data = {"price": Selector("10€"), "regularPrice": Selector("20€")}
```

Comment:
I was expecting this to work without `Selector`, but it does not. Should we consider making some of the processors in https://github.com/zytedata/zyte-parsers able to handle strings as inputs?

Reply:
I've forgot what were our design decisions with respect to returning strings from fields. @kmike ?

Did we want to make users disable the processor in such cases?

Reply:
Actually the design decisions I was thinking about were for the processors in zyte-common-items, not for lower-level functions in zyte-parsers, where it should be find to add text input support where possible.

Reply:
The support in zyte-parsers is added in https://github.com/zytedata/zyte-parsers/pull/11, but this code uses `MyProductFromListExtractor` for which the question stilll stands.